xattr.go

```diff
@@ -19,19 +27,36 @@ func (e *Error) Error() string {
 	return e.Op + " " + e.Path + " " + e.Name + ": " + e.Err.Error()
 }
 
-// Get retrieves extended attribute data associated with path.
+// Get retrieves extended attribute data associated with path. It will follow
+// all symlinks along the path.
 func Get(path, name string) ([]byte, error) {
+	return doGet(path, name, true)
```

Comment:
I suggest to refactor `doGet` - rename it to just `get` and instead of having `followSymlinks bool`, you can pass function reference (`getxattr` or `lgetxattr`) which you can call in `get`.
Those function may also return an extra `name` which we can pass to potential `Error{}`